cypress/e2e/tests/administration/jira-connection/negative.test.ts

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
This file is checking jira connection with wave.
Please rename it as interlinked.test.ts as in other modules.

Reply:
Fixed